packages/editor/src/components/post-publish-button/index.js

```diff
@@ -79,7 +79,7 @@ export class PostPublishButton extends Component {
 		};
 
 		const buttonProps = {
-			'aria-disabled': isButtonDisabled,
```

Comment:
I think the reason we prefer 'aria-disabled' is to avoid focus loss when the status of the button changes. So instead we do 'aria-disabled' and return early on the `onClick` handler.  Maybe we're missing the early return here.

Is this right @afercia ?

Reply:
I am not Andrea but yes that's right. We should be using `aria-disabled` and returning early from the `onClick` handler.

Reply:
From https://github.com/WordPress/gutenberg/pull/12073:

> From an accessibility perspective, using aria-disabled="true" and "noop'ing" an UI control is equivalent to disabling it with a disable attribute, as long as it is perceived as disabled by all users, i.e. also visually. The relevant difference is that a control with aria-disabled="true" is still focusable. Removing focusability from disabled elements can offer users both advantages and disadvantages. A must read recommendation can be found in the ARIA authoring practices:

Reply:
Late reply but yes: when there's the need to avoid a focus loss (as when a focused button gets disabled _while_ it's focused), it's possible to use `aria-disabled` and noop it. At that point, the only difference is that the button is still focusable. This can offer both advantages and disadvantages, to be evaluated on a case by case basis. For more details:

5.7 Focusability of disabled controls
https://www.w3.org/TR/wai-aria-practices-1.1/#kbd_disabled_controls